scripts/zones/Garlaige_Citadel/mobs/Hazmat.lua

```diff
@@ -6,6 +6,10 @@ require("scripts/globals/hunts")
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    mob:addMod(xi.mod.UFASTCAST, 50)
```

Comment:
couldn't find any mention of this little guy having fast cast. is this coming from wings?

Reply:
Got it from watching a guy fighting him on youtube. You can see that his flare is much faster than typical cast times.

https://youtu.be/B4_shxAGgDk?t=40